FBTweak/FBTweakInline.m

```diff
@@ -93,8 +94,8 @@ @implementation _FBTweakInlineLoader
 
 + (void)load
 {
-  static uint32_t _tweaksLoaded = 0;
-  if (OSAtomicTestAndSetBarrier(1, &_tweaksLoaded)) {
+  static atomic_bool _tweaksLoaded = 0;
```

Comment:
if it's a bool, use bool values.
also, you can do normal `dispatch_once.`

Reply:
Done.